GoogleForCreators/web-stories-wp
# Code Quality: Remove `enablePostLockingTakeOver` feature flag code

## Context

Now that the feature has been enabled by default for a while, we can safely remove this



## Summary

Updates cod to remove the feature flag



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions







This PR can be tested by following these steps:

1. Login a User 1
2. Create a story

In a new browser window
3. Login as User 2
4. Takeover the story

====

As user 1 note that user 2 has taken over

<img width="676" alt="Screen Shot 2022-11-29 at 7 32 55 AM" src="https://user-images.githubusercontent.com/62242/204530841-083e0434-6017-45fe-aead-1c81af746328.png">


<img width="750" alt="Screen Shot 2022-11-29 at 7 32 30 AM" src="https://user-images.githubusercontent.com/62242/204530858-58107e9b-4c6d-4d75-b3f0-939c83f988d8.png">



## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #12726
